### DISCUSSION TITLE
follow = 0 not working (FetchError: maximum redirect reached at: http://localhost:4444/)

### DISCUSSION THREAD

**Discussion by simllll** [node-fetch/node-fetch#1460] [General]
**Reproduction**

1. set follow to 0
2. call a 301 page
3. fetch error is thrown wiht maximum redirect reacher at

**Expected behavior**

no redirect should occur at all, like it is stated in the documentation, output should just return

**Your Environment**

| software         | version
| ---------------- | -------
| node-fetch       | 2.6.5
| node             | v16.10.0
| npm              | 7.24.1
| Operating System | linux

**Comment by simllll** [comment]
This is acutally the case for all limits, if I set "follow" to e.g. 3.. it's stopped by throwing an error, which makes it impossible to access the result.

**Comment by jimmywarting** [comment]
As i said in #1336 

> If i where able to decide then the `follow` option would not be available for use anymore... 
> if more granular control is needed then I think  a custom plugin should be using `manual`
>
> so that the code can work the same way with other fetch implementations in deno, node and the browser...

You also said you where fine with either way in #1336... so that is why i converted this to a discussion instead of having it as an issue (to be discussed about)
